tests/expressions/tests.py

```diff
@@ -496,7 +502,10 @@ def test_ticket_16731_startswith_lookup(self):
         )
         self.assertSequenceEqual(qs, [e2, e3])
 
-    def test_ticket_18375_join_reuse(self):
+    def test_f_works_with_multiple_relations_1(self):
```

Comment:
I don't like adding 1, 2, 3, 4
Every test tests that something "works" so I would change that naming
I would either say `f_expression` or remove `f` as all tests are for `F` here

Reply:
okay makes more sense.